barretenberg/cpp/src/barretenberg/commitment_schemes/zeromorph/zeromorph.hpp

```diff
@@ -648,7 +720,12 @@ template <typename Curve> class ZeroMorphVerifier_ {
                                       const std::vector<RefVector<Commitment>>& concatenation_group_commitments = {},
                                       RefSpan<FF> concatenated_evaluations = {})
     {
-        size_t log_N = multivariate_challenge.size();
+        FF log_N;
```

Comment:
need an issue here for bad code that just constructs a witness without linking it to circuit_size

Reply:
done